src/components/screens/monitor/delegates/forgingDetails.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { Link } from 'react-router-dom';
+import Box from '../../../toolbox/box';
+import BoxHeader from '../../../toolbox/box/header';
+import BoxContent from '../../../toolbox/box/content';
+import styles from './forgingDetails.css';
+import LiskAmount from '../../../shared/liskAmount';
+import { tokenMap, initialLSKSupply } from '../../../../constants/tokens';
+import routes from '../../../../constants/routes';
+
+const ForgingDetails = ({
+  t, networkStatus, latestBlock, nextForgers,
+}) => {
+  const totalForged = networkStatus && networkStatus.data.supply - initialLSKSupply;
+  const remainingBlocksInRound = latestBlock && 101 - (latestBlock.height % 101);
+  const nextForgersInRound = nextForgers.filter(
+    (forger, i) => i < 10 && i < remainingBlocksInRound,
+  );
```

Comment:
:nail_care: I think it would be simpler code and less computation to replace  `nextForgers.filter` with something `nextForgers.slice(0, Math.min(10, remainingBlocksInRound));`